README.md

```diff
@@ -2550,7 +2550,7 @@ Other Style Guides
     ;(() => {
       const name = 'Skywalker';
       return name;
-    }());
+    })();
```

Comment:
This would need an additional set of parens wrapping the entire IIAFE - the invoking parens and the function being invoked must be a single unit.